wipp-backend-data/src/main/java/gov/nist/itl/ssd/wipp/backend/data/imagescollection/images/ImageConversionService.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * This software was developed at the National Institute of Standards and
+ * Technology by employees of the Federal Government in the course of
+ * their official duties. Pursuant to title 17 Section 105 of the United
+ * States Code this software is not subject to copyright protection and is
+ * in the public domain. This software is an experimental system. NIST assumes
+ * no responsibility whatsoever for its use by other parties, and makes no
+ * guarantees, expressed or implied, about its quality, reliability, or
+ * any other characteristic. We would appreciate acknowledgement if the
+ * software is used.
+ */
+package gov.nist.itl.ssd.wipp.backend.data.imagescollection.images;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import javax.annotation.PostConstruct;
+
+import org.apache.commons.io.FilenameUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import gov.nist.itl.ssd.wipp.backend.core.CoreConfig;
+import gov.nist.itl.ssd.wipp.backend.data.imagescollection.ImagesCollectionRepository;
+import gov.nist.itl.ssd.wipp.backend.data.utils.flowjs.FlowFile;
+import gov.nist.itl.ssd.wipp.backend.data.utils.tiledtiffs.TiledOmeTiffConverter;
+import gov.nist.itl.ssd.wipp.backend.data.imagescollection.files.FileUploadController;
+import loci.common.services.DependencyException;
+import loci.common.services.ServiceException;
+import loci.formats.FormatException;
+
+
+/**
+*
+* @author Mohamed Ouladi <mohamed.ouladi at nist.gov>
+*/
+@Service
+public class ImageConversionService extends FileUploadController{
```

Comment:
This class should not be extending `FileUploadController` (because it is specific for the chunk upload with FlowJs). If some methods from `FileUploadController` are needed in several places, maybe they should be moved to a `FileUploadBase` class, and `ImageUploadBase`/`MetadataFileUploadBase` to handle the `getUploadSubFolder()` if needed